pkgs/development/tools/build-managers/shards/default.nix

```diff
@@ -1,22 +1,27 @@
-{ stdenv, fetchFromGitHub, crystal, pcre, libyaml, which }:
+{ stdenv, fetchFromGitHub, crystal }:
 
 crystal.buildCrystalPackage rec {
   pname = "shards";
   version = "0.10.0";
 
   src = fetchFromGitHub {
-    owner  = "crystal-lang";
-    repo   = "shards";
-    rev    = "v${version}";
-    sha256 = "1bjy3hcdqq8769bx73f3pwn26rnkj23dngyfbw4iv32bw23x1d49";
+    owner = "peterhoeg";
```

Comment:
Are you the new maintainer of `shards`?

Reply:
Ah. This is the vendored version. Maybe [nix-community](https://github.com/nix-community) would be a better place for this, so we can add more contributors easier.

Reply:
I actually think we can do without the vendored version. Let me take a look.

Reply:
And we can. Plus we now get man pages as well.